cds-snc/cppd-medical-report-sails
# Update package dependencies

# What this does

Updated package dependencies. This includes TailwindCSS 1.2, which includes a bunch of cool new stuff like grids, transitions, and transforms!

## To test
This should not affect anything, but useful to take a quick tour around to make sure nothing's broken.
- pull in this branch
- run `npm install` to get latest everything
- test all the things
